wine-tkg-git/customization.cfg

```diff
@@ -193,6 +193,9 @@ _stg_shared_mem_default="true"
 # Update winevulkan to whatever version I have pushed last, til next time, for new shiny and tasty vk extensions support. Thanks dadドイツ人 !
 _update_winevulkan="true"
 
+# Use the DXVK "Async" patch. WARNING: may cause anti-cheat systems to go off.
```

Comment:
Moving this next to other dxvk related options would make sense

Reply:
Done.